test/README.md

```diff
@@ -150,8 +162,10 @@ In-test variables
                          Optionally a MODEL suffix can used for further filtering, e.g.
                          win32 win64 linux32 linux64 osx32 osx64 freebsd32 freebsd64
 
-Relevant environment variables
+Environment variables
 ------------------------------
```

Comment:
FWIW I tried to align the MarkDown header, but if it's meh to you, maybe use `##` ?